core/modules/obsolete/TableMerger.cc

```diff
@@ -102,9 +104,9 @@ void inplaceReplace(std::string& s, std::string const& old,
     if(rplc.size() < rplcSize) { // do padding for in-place
         rplc += std::string(rplcSize - rplc.size(), ' ');
     }
-    //LOGF_DEBUG("rplc %1% old=%2%" % rplc % s.substr(pos-1, rplcSize+2));
+    //LOGS(_log, LOG_LVL_DEBUG("rplc " << rplc << " old=" << s.substr(pos-1, rplcSize+2));
```

Comment:
This will not compile if you uncomment it. Maybe worth deleting commented-out code entirely? Same thing couple of lines below.